packages/odata-common/src/request-builder/action-function-import-request-builder-base.ts

```diff
@@ -3,13 +3,15 @@ import { HttpResponse } from '@sap-cloud-sdk/http-client';
 import { v4 as uuid } from 'uuid';
 import { ODataRequestConfig } from '../request/odata-request-config';
 import { BatchReference, WithBatchReference } from '../request';
+import { DeSerializers } from '../de-serializers/de-serializers';
 import { MethodRequestBuilder } from './request-builder-base';
 
 /**
  * Create OData request to execute a action or function import.
  * @typeParam ReturnT - Type of the function import return value.
  */
 export abstract class ActionFunctionImportRequestBuilderBase<
+    DeSerializersT extends DeSerializers,
```

Comment:
[q] This is actually a breaking change. Maybe we should add this as the last type parameter and make it optional + add a todo to change this in the future and give it a compatibility note.
However, this will make things a bit more complicated, so I am wondering whether this is worth it. I doubt anyone is using this directly, therefore maybe it is fine with a compatibility note.

Reply:
[req] Please also add the typeParam to the API Doc